src/CommandExecution.ts

```diff
@@ -1,28 +1,42 @@
-import { Client, Message } from "discord.js";
+import { Client } from "discord.js";
 import { Command, CommandType, ImpersonateCommand } from "./CommandParser";
 import { ImpersonationHandler } from "./Impersonation";
 
+/*
+  A single interface to contain all of the implementions for each command.
```

Comment:
could you clarify why we need this? Can be in comments or on this thread

Reply:
You are right, we don't. Commands themselves now have an execute function and the CommandExecution has been removed entirely.